.eslintrc.json

```diff
@@ -14,7 +14,8 @@
         "inject": false,
         "module": false,
         "Sk": false,
-        "tie": false
+        "tie": false,
+        "tieBasic": false
```

Comment:
Oh, yeah, that's my bad. I was going to say tieCommon or tieConfig; do you have a preference?